python_requirements

```diff
@@ -1,3 +1,4 @@
+wheel
```

Comment:
Why do we need this dependency explicitly specified? Not opposed, just curious.

Reply:
IDK, somehow it doesnt work on a clean install.